tests/unit/test_credentials_utils.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+from hammr.utils import credentials_utils
+from hammr.utils.credentials_utils import CredentialException
+from mock import Mock,patch,MagicMock
+
+__author__ = 'UShareSoft'
+
+
+class TestCredential(TestCase):
+
+    @patch('json.load')
+    @patch('hammr.utils.credentials_utils.check_credfile')
+    @patch('__builtin__.open')
+    def test_fill_credentials_from_credfile(self, open, check_credfile, load):
+        #Given
+        cred = credentials_utils.Credential()
+        open.return_value = MagicMock(spec=file)
+        check_credfile.return_value = 'credential.json'
+        load.return_value = {'url' : 'my_url', 'user' : 'my_user', 'publicKey' : 'my_publicKey', 'secretKey' : 'my_secretKey', 'acceptAutoSigned' : 'True'}
```

Comment:
you should extract those strings into variables, you'll be able to reuse them in assertion